src/EditorFeatures/Core/Implementation/BraceMatching/ClassificationTypeDefinitions.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
+#pragma warning disable 649
```

Comment:
Then do we need to move the build earlier so the WPF build also picks it up? This should be fixable.